### PR TITLE
Closes AgileVentures/MetPlus_tracker#291

### PR DESCRIPTION
**Pull request by patmbolger**

### CODE REVIEW

File: app/views/devise/invitations/new.html.haml [hunk 1]
```diff
@@ -1,6 +1,10 @@
 %h2= t "devise.invitations.new.header"
 = form_for resource, :as => resource_name, :url => invitation_path(resource_name), :html => {:method => :post} do |f|
-  = devise_error_messages!
+  -# = devise_error_messages!
```
Comment (joaopapereira) [inline app/views/devise/invitations/new.html.haml]: We should remove this if we are not going to use it

File: spec/controllers/agencies_controller_spec.rb [hunk 2]
```diff
@@ -65,12 +65,9 @@
         @agency.valid?
         agency_admin = FactoryGirl.create(:agency_admin, agency: @agency)
         sign_in agency_admin
-        patch :update, agency: FactoryGirl.attributes_for(:agency, 
+        patch :update, agency: FactoryGirl.attributes_for(:agency,
                         phone: '', website: 'nodomain'), id: @agency
       end
-      it 'assigns @model_errors for error display in layout' do
```
Comment (joaopapereira) [inline spec/controllers/agencies_controller_spec.rb]: Maybe just removing this is not enough, are we checking the error messages are set? Or that we are rendering something?

**Comment by joaopapereira** [comment]
The comment about the specs need to be revisited for all the specs

**Comment by joaopapereira** [comment]
Looks good, please review my comments

**Comment by patmbolger** [comment]
Changes made in response to comments.

**Pull request merged**